tests/actions/test_action.py

```diff
@@ -2,6 +2,7 @@
 from unittest.mock import MagicMock
 
 import pytest
+from secureli.abstractions.echo import Color
```

Comment:
Can we just move the Color enum to a new file `./secureli/models/ansi_term_colors.py`?
And in there we can also link to this page to explain were this list is coming from/what the valid values can be: https://en.wikipedia.org/wiki/ANSI_escape_code#Colors